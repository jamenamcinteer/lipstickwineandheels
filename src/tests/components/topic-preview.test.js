import React from 'react'
import { shallow } from 'enzyme'
import TopicPreview from '../../components/topic-preview'
import data from '../fixtures/data'

test('should render article preview component correctly', () => {
  const wrapper = shallow(<TopicPreview topic={data.data.contentfulTag} />)
  expect(wrapper).toMatchSnapshot()
})
