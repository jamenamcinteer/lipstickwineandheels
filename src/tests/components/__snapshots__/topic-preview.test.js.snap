// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render article preview component correctly 1`] = `
<div
  className="preview"
>
  <div
    className="imageContainer"
  >
    <GatsbyLink
      to="/tag/tech"
    >
      <Image
        Tag="div"
        alt="this is the image title"
        fadeIn={true}
        sizes={
          Object {
            "src": "/reference/to/some/image.jpg",
          }
        }
      />
      <h3
        className="topicTitle"
      >
        Tech
      </h3>
    </GatsbyLink>
  </div>
</div>
`;
